composer.json

```diff
@@ -27,15 +27,15 @@
     },
     "extra": {},
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```

Comment:
Do we still need version 8 here?

Reply:
As a migration tool, I thought it would be beneficial to support a wider range and only drop versions when there was some other requirement - Happy to drop it though…

Reply:
There are no changes in the latest versions, so anyone using PHP 8.0 can use an older version. Migration to a newer version of Mezzio will not work anyway.
Or I am wrong?

Reply:
You're right - users can always use an older version - I'll drop it. Gimme a sec